zerver/webhooks/github/view.py

```diff
@@ -574,13 +607,11 @@ def api_github_webhook(
 
     body_function = get_body_function_based_on_type(event)
 
-    if 'include_title' in signature(body_function).parameters:
-        body = body_function(
-            payload,
-            include_title=user_specified_topic is not None,
-        )
-    else:
-        body = body_function(payload)
+    helper = Helper(
```

Comment:
Maybe a name like GitHubParams or BodyParams would make sense

Reply:
I am sticking with Helper for now, and we can rename this later.

Reply:
(It actually does evolve to more than `BodyParams` now.)